taxonomies/additives.txt

```diff
@@ -2666,17 +2671,21 @@ de:E162, Betanin, Beetenrot, Betanoin, Betenrot
 el:E162, Ερυθρα χρωστικη της ριζας των τευτλων, βετανινη
 es:E162, Rojo de remolacha, Betanina, Betacianina, E 162, Rojo remolacha
 et:E162, Peedipunane, betaniin
+fa:E162, بتانین
 fi:E162, Punajuuriväri, betalaiini
-fr:E162, Rouge de betterave, bétanine, rouge betterave, E 162, 7659-95-2, C24H27N2O13
+fr:E162, Rouge de betterave, bétanine, rouge betterave, EBetterave rouge
```

Comment:
```suggestion
fr:E162, Rouge de betterave, bétanine, rouge betterave
```
"betterave rouge" means "red beetroot", it's not an additive like "rouge de betterave" (beetroot red), it's the actual vegetable.